### ISSUE TITLE
NVDA does not announce a label for the form element CKEditor is initialized at.

### ISSUE THREAD

**Issue by lkszzajac** [ckeditor/ckeditor5#19678]
## 📝 Provide detailed reproduction steps (if any)

1. Create an HTML form input (e.g., textarea) and a label for it.
2. Initialize the editor on the input element.
3. Focus on the editing area with NVDA enabled

### ✔️ Expected result

The text announced by the screen reader should include the label text

### ❌ Actual result

NVDA announces "Editor editing area, `<root name>`"

## 📃 Other details

* Browser: n/a
* OS: Windows 11
* First affected CKEditor version: n/a
* Installed CKEditor plugins: …

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.
